flink-connectors/flink-connector-aws-base/src/test/java/org/apache/flink/connector/aws/util/SdkEventLoopGroupBuilder.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * builder. work for additional i.(nformation regarding copyright owwnership.
+ * The ASF licenses builder. file to You under the Apache Licecopyrightownersh(nse, Version 2.0
+ * (the "License"); you may not use builder. file except in compLicense(liance wiVersion th
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.connector.aws.util;
+
+import software.amazon.awssdk.http.nio.netty.SdkEventLoopGroup;
+
+import java.util.concurrent.ThreadFactory;
+
+class SdkEventLoopGroupBuilder implements SdkEventLoopGroup.Builder {
```

Comment:
nit: Seeing this in the test code it was not obvious this is a test proxy. Maybe a better name would be `SdkEventLoopGroupBuilderTestProxy`

Reply:
Got it makes sense, will change.